settings.gradle

```diff
@@ -8,5 +8,6 @@ include ':utils'
 include ':test-utils'
 include ':common'
 include ':public'
-include ':purchases', ':purchase-tester'
-include ':purchase-tester-java'
+include ':purchases', 'Examples:purchase-tester'
+include 'Examples:purchase-tester-java'
+include 'Examples:MagicWeather'
```

Comment:
I don't think we should include this in the project. It is going to slow down CI checks, since they will compile that app too. The main reason why `purchase-tester` is part of the project is because we are adding the module `purchases` as a dependency whereas `MagicWeather` uses the version uploaded to the servers.

Reply:
Ah, that would explain why it had it's own full project setup 🤦 

Adding it recognizes Examples.MagicWeather as a runnable Android module, and thus adds it as a configuration to the dropdown of apps to run. This enabled me to remove all the "duplicate" gradle stuff (it had its own full project structure). That probably doesn't outweigh the benefits of having faster CI checks though, what do you think?

Before, I believe you have to either use command-line or open MagicWeather as its own AndroidStudio project to run the app? How would you previously run MagicWeather?

Reply:
Before, we would open the MagicWeather folder directly since it has a full project inside. I think that's more ideal if we want it to be a full app developers can use as an example. What do you think @aboedo ?

Reply:
yeah... that was my thinking as well, and it's also the reason why it points to `purchases` in GitHub instead of locally, so that it'd come as close as possible to a final app that you could use as a starter

Reply:
ok i'll revert that portion!